packages/state-utils/src/create-selector/index.ts

```diff
@@ -1,12 +1,12 @@
 /**
  * External dependencies
  */
-import { memoize, includes } from 'lodash';
-import isShallowEqual from '@wordpress/is-shallow-equal';
+import { memoize } from 'lodash';
```

Comment:
I thought about this but ultimately landed on using lodash's because we don't currently employ memize anywhere in the codebase but do use lodash's memoize. I didn't want to try to introduce a new memoization library as well as doing this extraction.